views/js/review/proxy/qtiTestReviewProxy.js

```
@@ -58,8 +58,12 @@ define([
              * @returns {Promise}
              */
             this.request = (url, reqParams, contentType, noToken) => {
+                let _url = url;
+                if(true) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)